dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/deprecated/tracker/enrollment/AbstractEnrollmentService.java

```diff
@@ -1541,7 +1543,8 @@ private void saveTrackedEntityComment(
 
         comment.setCreated(created);
 
-        comment.setLastUpdatedBy(user);
+        // TODO is this needed here?
+        comment.setLastUpdatedBy(userService.getUser(user.getUid()));
```

Comment:
This is the only place where an actual `User` is needed to set the last updated field but I think we maybe have a better solution and this would happen anyway in the store? @netroms